plugins/ccgx/fu-ccgx-hpi-device.c

```diff
@@ -1459,10 +1458,6 @@ fu_ccgx_hpi_device_set_quirk_kv (FuDevice *device,
 				     "invalid ImageKind");
 		return FALSE;
 	}
-	g_set_error_literal (error,
-			     G_IO_ERROR,
-			     G_IO_ERROR_NOT_SUPPORTED,
-			     "no supported");
```

Comment:
this needs to remain

Reply:
I fixed it. 
Please check [https://github.com/fwupd/fwupd/pull/2013/commits/605b1fce02e8d9e66e7e31c790f0d82ce6635827](url)